src/acquisition/covidcast/covidcast_meta_cache_updater.py

```diff
@@ -15,6 +15,7 @@ def get_argument_parser():
 
   parser = argparse.ArgumentParser()
   parser.add_argument("--log_file", help="filename for log output")
+  parser.add_argument("--num_threads", type=int, help="number of worker threads to spawn for processing of each source/signal pair")
```

Comment:
(number of threads for the group of pairs, not number of threads per pair)
```suggestion
  parser.add_argument("--num_threads", type=int, help="number of worker threads to spawn for processing source/signal pairs")
```